README.md

```diff
@@ -1,10 +1,20 @@
+---
+version: 0.7.0-SNAPSHOT
+---
+
 # Template Webpack REST Server
 
+[![GitHub License](https://img.shields.io/github/license/devlinjunker/template.hapi.rest)](https://github.com/devlinjunker/template.hapi.rest/blob/master/LICENSE)
+[![CII Best Practices Summary](https://img.shields.io/cii/summary/4288?label=core-infrastructure)](https://bestpractices.coreinfrastructure.org/en/projects/4288)
+[![GitHub last commit](https://img.shields.io/github/last-commit/devlinjunker/template.hapi.rest)](https://github.com/devlinjunker/template.hapi.rest/commits/master)
+
```

Comment:
add more badges?
- [ ] npm dependencies
- [ ] npm/node versions ([specify in package.json](https://docs.npmjs.com/files/package.json#engines))
- [ ] swagger/openapi
- [ ] build status?
- [ ] tests passing? (need to add [workflow](https://help.github.com/en/actions/configuring-and-managing-workflows/configuring-a-workflow#adding-a-workflow-status-badge-to-your-repository) to run tests)
- [ ] Code Quality? app.codacy.com
- [ ] Maintainability? codeclimate.com